charts/scorpio-broker-dist/values.yaml

```diff
@@ -0,0 +1,982 @@
+   initialDelaySeconds: 40
+   failureThreshold: 6
+   periodSeconds: 10      
+ 
+## Pod Security Context
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+##
+  securityContext: {}
+  
+## Pod restart policy
+  restartPolicy: Always
+
+## Pod Service Account
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+  serviceAccount:
+   enabled: false
+  ## Name of an already existing service account. Setting this value disables the automatic service account creation.
+   name: ""
+## ref: https://pracucci.com/graceful-shutdown-of-kubernetes-pods.html
+  terminationGracePeriodSeconds: 30
+
+## Enable HPA for eureka
+## ref: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+##
+  hpa:
+   enabled: true
+   # --  minimum number of replicas to which the autoscaler can scale down
+   minReplicas: 1
+   # -- maximum number of pods that can be set by the autoscaler
+   maxReplicas: 5
+   # --  average CPU usage across all the pods exceeds 80%, HPA will spin up additional pods.
+   targetCPUUtilizationPercentage: 80
+
+################################### Gateway ##########################################################################
+gateway:
+## Enable the gateway deployment & service
+  enabled: true 
+  name: gateway
+## Custom label value
+  gatewayLabel: gateway  
+## Number of gateway instances to deploy
+##
+  replicas: 1
+
+  updateStrategy:
+    # -- type of the update
+    type: RollingUpdate
+    # -- new pods will be added gradually
+    rollingUpdate:
+      # -- number of pods that can be created above the desired amount while updating
+      maxSurge: 25%
+      # -- number of pods that can be unavailable while updating
+      maxUnavailable: 25%
+## Scorpio Broker gateway image version
+## ref: https://hub.docker.com/r/scorpiobroker/scorpio
+## Please use only latest version of docker images as few older images might not work with kubernetes.
+  image:       
+   repository: scorpiobroker/scorpio
+   tag: gateway_1.16.0
+## Specify a imagePullPolicy
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+   pullPolicy: Always
+
+## gateway containers' resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+##
+  resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to use HPA or specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits: 
+  #   cpu: 250m
+  #   memory: 1Gi
+  # requests: 
+  #   cpu: 250m
+  #   memory: 256Mi
+  
+## gateway containers'  readiness probes. Evaluated as a template.
+## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+##
```

Comment:
Thats not an address, its a path. Since this uses $HOSTNAME (which I suppose is an env var), where is this configured? The containers usually do not have a hostname.

Reply:
Hi,
This $HOSTNAME is not configured in env var as it is resolving into the IP address of the pod itself.